theseus/geometry/se3.py

```diff
@@ -285,14 +285,26 @@ def exp_map(
 
             jac[:, 3:, 3:] = jac[:, :3, :3]
 
+            minus_one_by_twelve = torch.full(
+                near_zero.shape,
+                -1 / 12.0,
+                dtype=sine_by_theta.dtype,
+                device=sine_by_theta.device,
+            )
             d_one_minus_cosine_by_theta2 = torch.where(
                 near_zero,
-                -1 / 12.0,
+                minus_one_by_twelve,
```

Comment:
Can you add this commit as a separate PR to main so the fix is disentangled from this PR and other PRs can immediately rebase on (once merged to main you can rebase here as well).